Author: Pino Toscano
Description: sys/soundcard.h includes linux/soundcard.h on linux,
 so there is no need for complications
Forward: https://sourceforge.net/tracker/?func=detail&aid=3297661&group_id=55736&atid=478072
--- puredata.orig/src/s_audio_oss.c
+++ puredata/src/s_audio_oss.c
@@ -5,11 +5,7 @@
 
 /* this file inputs and outputs audio using the OSS API available on linux. */
 
-#if defined(__FreeBSD_kernel__)
-# include <sys/soundcard.h>
-#else
-# include <linux/soundcard.h>
-#endif
+#include <sys/soundcard.h>
 
 #ifndef SNDCTL_DSP_GETISPACE
 #define SNDCTL_DSP_GETISPACE SOUND_PCM_GETISPACE
